torch/_inductor/codegen/wrapper.py

```diff
@@ -558,6 +563,11 @@ def __init__(self):
         self._call_func_id = next(CppWrapperCodeGen.call_func_id)
         super().__init__()
 
+    def set_output_refs(self):
+        self.output_refs = [
+            x.cpp_wrapper_codegen_reference() for x in V.graph.graph_outputs
```

Comment:
Maybe we can check the existence of `cpp_wrapper_codegen_reference` and call it or call `codegen_reference` instead here? With this, you don't have to add `cpp_wrapper_codegen_reference` to all the related IR nodes.

Reply:
Fixed as suggested.